composer.json

```diff
@@ -14,17 +14,22 @@
 		"squizlabs/php_codesniffer": "^2.9.0 || ^3.0.2",
 		"wp-coding-standards/wpcs": "*"
```

Comment:
Also checked with `dev-develop`, which is as close to `1.0.0` as it can be, and no new violations are reported :+1:

Reply:
+1